tests/tensor_search/test_get_settings_backwards_compatibility.py

```diff
@@ -0,0 +1,101 @@
+import unittest
+from unittest.mock import MagicMock, patch
+from fastapi.testclient import TestClient
+from marqo.tensor_search import api
+from marqo.core.models.marqo_index import (
+    StructuredMarqoIndex, UnstructuredMarqoIndex, TextPreProcessing, ImagePreProcessing,
+    HnswConfig, Model, DistanceMetric, VectorNumericType, TextSplitMethod
+)
+
+class TestGetSettingsBackwardsCompatibility(unittest.TestCase):
+    def setUp(self):
+        self.mock_config = MagicMock()
+        self.mock_index_management = MagicMock()
+        self.mock_config.index_management = self.mock_index_management
+        api.app.dependency_overrides[api.get_config] = lambda: self.mock_config
+        self.client = TestClient(api.app)
+
+    def tearDown(self):
+        api.app.dependency_overrides.clear()
+
+    def create_mock_index(self, index_class, **kwargs):
+        common_params = {
+            "name": "test_index",
+            "schema_name": "test_schema",
+            "model": Model(name="test_model", properties=None),
+            "normalize_embeddings": True,
+            "text_preprocessing": TextPreProcessing(
+                split_length=100,
+                split_overlap=0,
+                split_method=TextSplitMethod.Word
+            ),
+            "image_preprocessing": ImagePreProcessing(),
+            "video_preprocessing": None,
```

Comment:
You won't get None for <2.12. They'll be missing

Reply:
I've removed the audio and video preprocessing: None lines and confirmed that tests pass